components/service/glean/src/main/java/mozilla/components/service/glean/storages/StorageEngine.kt

```diff
@@ -20,4 +20,11 @@ internal interface StorageEngine {
      *         ancestor is [Object], so we need to return [Any].
      */
     fun getSnapshotAsJSON(storeName: String, clearStore: Boolean): Any?
+
+    /**
+     * Indicate whether this storage engine is sent at the top level of the ping
+     * (rather than in the metrics section).
+     */
+    val sendAsTopLevelField: Boolean
+        get() = false
```

Comment:
See above

Reply:
Unfortunately with
```
    val sendAsTopLevelField: Boolean = false
```
it says `Property initializers are not allowed in interfaces`.

Reply:
Ops, yes, makes sense. This should only work in `ExperimentsStorageEngine`.